### DISCUSSION TITLE
联想智选 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#69] [模板讨论]
联想智选 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by xueliymd** [comment]
你好请问一下这个报错是什么原因阿
Failed at 8/14 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 400: Bad Request,
Request URL: https://api.club.lenovo.cn/users/getSessionID?s={{s|urlencode}}

**Reply by wjf0214** [reply]
之前网站的问题吧。我那两天也有问题，现在看正常了。
